.github/workflows/integrate.yaml

```diff
@@ -55,7 +55,21 @@ jobs:
       - uses: actions/checkout@v3
       - run: python3 -m pip install tox
       - run: tox -e ${{ matrix.charm }}-unit
-  
+
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    strategy:
+      matrix:
+        charm:
+          - katib-controller
+          - katib-db-manager
+          - katib-ui
+    with:
+      charm-path: ./charms/${{ matrix.charm }}
+      model: kubeflow
```

Comment:
Katib can be deployed in any namespace, we can use the default one from the wf.

Reply:
we cannot use the default one from the wf i.e. `testing` because the `Setup opreator environment` step in `Apply Terraform` workflow is creating a `testing` model, then the `Create juju model` step is also trying to create a model.
I discussed with @mvlassis in https://github.com/canonical/envoy-operator/pull/134 and he's filing an issue against the terraform workflow, since we don't need to create 2 models.